Automattic/social-logos
# Build: add back svg-min files

Follow up to https://github.com/Automattic/gridicons/pull/191, this adds back the svg-min files, and moves react preprocessing into a tmp folder that gets cleaned up after the build runs.

Changes also include whitelisting exact files in package.json so we can remove the .npmignore

## Testing Instructions
- `npm run build` 
- This should generate no local changes for git
- Adding an svg then running `npm run build` works normally
- temporary svg-min-react folder is cleaned up
- svgs in svg-min folder do not have transformed kebab case to camel case attributes.
Eg: `svg-min/eventbrite.svg` has `fill-rule` instead of `fillRule`
- Generated react components do have camel case. eg search for `fillRule` in `build/index.js`
